ffi/saltyrtc_task_relayed_data_ffi.h

```diff
@@ -478,6 +515,27 @@ salty_client_connect_success_t salty_client_connect(const salty_handshake_future
                                                     const salty_channel_sender_rx_t *sender_rx,
                                                     const salty_channel_disconnect_rx_t *disconnect_rx);
 
+/*
+ * Decrypt raw bytes using the session keys after the handshake has been finished.
+ *
+ * Note: The returned data must be explicitly freed with
+ * `salty_client_encrypt_decrypt_free`!
```

Comment:
Question: Does every function that allocates have its own free function and is there a technical need for it?

Reply:
In this case, the free function is shared for encryption and decryption functions.

Yes, in most cases having separate free functions is technically required, as the freeing/dereferencing depends on the type being freed. For example, sometimes nested types need to be reconstructed separately for them to be freed on drop. (Remember, in Rust you don't allocate/deallocate a block of memory, you allocate and free implicitly by taking and dropping ownership of a value.